README.md

```diff
@@ -40,6 +40,24 @@ PROVIDER_URL=url of the OIDC provider
 Or supply the `PROVIDER_URL` option setting in an environment variable called `KOLIBRI_OIDC_CLIENT_URL`.
 
 
+### OIDC endpoints
+In case some of the endpoints returned by the OIDC discovery url `.well-known/openid-configuration` are not standard, you can set them using these options either in the `$KOLIBRI_HOME/options.ini` file or suplying them in an environment variable.
```

Comment:
```suggestion
### OIDC endpoints

In case some of the endpoints returned by the OIDC discovery url `.well-known/openid-configuration` are not standard, you can set them using these options either in the `$KOLIBRI_HOME/options.ini` file or by supplying them in an environment variable.

In the `options.ini` file:

```